pkg/flash/flash_linux.go

```diff
@@ -52,37 +45,58 @@ func New(spi SPI) (*Flash, error) {
 		spi: spi,
 	}
 
+	// no errors.Join in ci yet.
+	// For now, ignore errors in SFDPReader until
+	// we get a part that can even provide
+	// a correct implementation.
```

Comment:
I tested a macronix part with this implementation and it works.

OpenTitan also supports SFDP instruction, see https://opentitan.org/book/hw/ip/spi_device/

Reply:
great. SFDP is our future, I'm impressed you made it work, it seems so complex!